engine/pinball_game/end_of_ball_bonus.asm

```diff
@@ -543,15 +543,15 @@ Func_f8d5: ; 0xf8d5
 	ret
 
 AddBigBCD6: ; 0xf902
-x = 0
+DEF x = 0
 rept 6

```

Comment:
lol this should just be a for loop now

Reply:
Easy enough; i'll take care of it when I get a chance tonight.